app/src/main/java/fr/free/nrw/commons/navtab/MoreBottomSheetFragment.kt

```diff
@@ -170,6 +181,28 @@ class MoreBottomSheetFragment : BottomSheetDialogFragment() {
     fun uploadFeedback(feedback: Feedback) {
         val feedbackContentCreator = FeedbackContentCreator(requireContext(), feedback)
 
+        if (!isInternetConnectionAvailable(requireContext())) {
```

Comment:
The issue doesn't lies when you are communicating between client & server. The issue is we are not handling failing of network call when the problem arises due to low/no data connection, if I am not wrong.